apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -0,0 +1,91 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {Heading1, Heading2, Heading3} from '@cdo/apps/lib/ui/Headings';
+import style from './school-association.module.scss';
+import SimpleDropdown from '../componentLibrary/simpleDropdown/SimpleDropdown';
+import {COUNTRIES} from '@cdo/apps/geographyConstants';
+import SearchBar from './SearchBar';
+
+const TEST_SCHOOL_DATA = [{value: 'test', text: 'TestSchool'}];
```

Comment:
just checking -- this is just here for now right?

Reply:
Yes! Just for me to see the UI clearly before I actually have to get the zipcode field filtering all the schools